### PR TITLE
[Transalation [Indonesian]: /id/guide/filter-queries.md

### PR DESCRIPTION
**Pull request by CikiMomogi**
Ready to review.

Phew, this one took me a while to work on. The Query Language section was enough to rack my brain. There are a lot of technical terms that I have never heard of. However, It may still be able to be improved, especially the Query Language section.

To do:
- [x] Translation
- [x] https://github.com/grammyjs/website/pull/411

### CODE REVIEW

File: site/docs/id/guide/filter-queries.md [hunk 1]
```diff
@@ -3,96 +3,97 @@ prev: ./api.md
...
 
 ```ts
 bot.on(":text"); // Seluruh pesan dan postingan channel yang mengandung teks
 bot.on("message::url"); // Pesan yang mengandung URL baik di teks maupun di caption (foto, dan sebagainya)
 bot.on("::email"); // Seluruh pesan dan postingan channel yang mengandung email di teks maupun di caption-nya
 ```
 
-Mengosongkan nilai _pertama_ akan menyocokkan pesan serta postingan channel.
-[Perlu diingat](./context.md#aksi-yang-tersedia) bahwa `ctx.msg` memberi kamu akses ke pesan dan postingan channel manapun yang cocok dengan kueri yang diberikan.
+Mengosongkan value _pertama_ akan menyocokkan pesan serta postingan channel.
```
Comment (sensen) [inline site/docs/id/guide/filter-queries.md:74]:
```suggestion
Mengosongkan value _pertama_ akan mencocokkan pesan serta postingan channel.
```

File: site/docs/id/guide/filter-queries.md [hunk 2]
```diff
@@ -3,96 +3,97 @@ prev: ./api.md
...
+[Perlu diingat](./context.md#aksi-yang-tersedia) bahwa `ctx.msg` memberi kamu akses ke pesan dan postingan channel manapun yang cocok dengan query yang diberikan.
 
-Mengosongkan nilai _kedua_ menyocokkan entitas pesan serta entitas caption.
+Mengosongkan value _kedua_ akan menyocokkan entity, baik di pesan maupun di caption.
```
Comment (sensen) [inline site/docs/id/guide/filter-queries.md:77]:
```suggestion
Mengosongkan value _kedua_ akan mencocokkan entity, baik di pesan maupun di caption.
```

**Review by sensen — COMMENTED** [review]
Ketemu 2 typo ini aja.

**Comment by sensen** [comment]
Haha.. Thank you! I feel so special!

File: site/docs/id/guide/filter-queries.md [hunk 3]
```diff
@@ -0,0 +1,359 @@
...
+
+```ts
+bot.on("chat_member");
+```
+
+yang digunakan untuk mendeteksi perubahan status member chat lainnya, misalnya ketika seseorang bergabung, dipromosikan, dan sebagainya.
+
+> Ingat! Update `chat_member` perlu diaktifkan secara eksplisit dengan cara menentukan `allowed_updates` saat memulai bot kamu.
+
+### Mengombinasikan Query sengan Method Lain
```
Comment (fadzikri) [inline site/docs/id/guide/filter-queries.md:223]:
```suggestion
### Mengombinasikan Query dengan Method Lain
```

**Review by fadzikri — CHANGES_REQUESTED** [review]
Typo

**Comment by fadzikri** [comment]
> Phew, this one took me a while to work on. The Query Language section was enough to rack my brain. There are a lot of technical terms that I have never heard of. However, It may still be able to be improved, especially the Query Language section.

Yeah that's great one work, i can read that documentation easily and the terms are friendly enough to understand.

**Review by fadzikri — APPROVED** [review]
Ok, good ...

**Pull request merged**